packages/dev/core/src/FlowGraph/Blocks/Data/flowGraphCoordinateTransformBlock.ts

```diff
@@ -0,0 +1,45 @@
+import { FlowGraphBlock } from "../../flowGraphBlock";
+import type { FlowGraphContext } from "../../flowGraphContext";
+import type { FlowGraphDataConnection } from "../../flowGraphDataConnection";
+import { RichTypeAny, RichTypeVector3 } from "../../flowGraphRichTypes";
+import { Vector3 } from "../../../Maths";
+import type { TransformNode } from "../../../Meshes/transformNode";
+
+/**
+ * @experimental
+ * This blocks transforms a vector from one coordinate system to another.
+ */
+export class FlowGraphCoordinateTransformBlock extends FlowGraphBlock {
+    public readonly sourceSystem: FlowGraphDataConnection<TransformNode>;

```

Comment:
I wonder if this should not be 2 blocks ???
1. A FlowGraphCoordinateTransformBlock that only transforms with one matrix input
2. A block to extract the conversion matrix from one space to another one

Reply:
We'd have to create these two blocks every time we parsed from GLTF and encountered this block, is that ok?

Reply:
So we could have three cause in a lot of cases, we have a matrix to transform smthg. I think they are not mutually exclusive and keeping all the blocks from the spec might be a good idea.